hr_attendance_management/README.rst

```diff
@@ -46,6 +46,14 @@ To configure this module, go in **Attendances/Configuration/Attendance Settings*
 #. In **Leaves/Configuration/Leave Types** configure if leave types should not deduce from the due hours of
    the employees by setting *Keep due hours*
 
+To configure the cron who update the balance annually , go in **Settings/Technical/Automation/Automated Actions**:
+
+#. In the cron *Compute annual extra hours balance* choose the date you want to apply the process to move the extra hours in the annual balance.
+   This process aime to avoid to compute a huge amount of data after few years.
```

Comment:
This process avoids the system to compute a huge amount of data after a few years of attendance.